.eslintrc.json

```diff
@@ -47,7 +47,7 @@
     "quotes": [2, "single"],
     "semi": [2, "always"],
     "space-before-blocks": 2,
-    "space-in-parens": [2, "never"],
+    "space-in-parens": [0, "never"],
```

Comment:
Why ? It's because the following wrapping rule ? `if ( k === 'something')`

Reply:
yup